diesel/src/query_dsl/save_changes_dsl.rs

```diff
@@ -50,3 +50,25 @@ impl<T, ST> SaveChangesDsl<SqliteConnection, ST> for T where
         T::table().find(self.id()).get_result(conn)
     }
 }
+
+#[cfg(feature = "mysql")]
+use mysql::{MysqlConnection, Mysql};
+#[cfg(feature = "mysql")]
+use associations::Identifiable;
+
+#[cfg(feature = "mysql")]
+impl<T, ST> SaveChangesDsl<MysqlConnection, ST> for T where
```

Comment:
This is literally the same code as the sqlite impl, but I don't think it's worth making a macro for 2 cases.